src/ast.cpp

```diff
@@ -27,6 +29,15 @@ Token const* Identifier::token() const {
 	return static_cast<CST::Identifier*>(m_cst)->m_token;
 }
 
+
+SequenceExpression* convert_and_wrap_block_in_seq_expr(CST::Block* cst, Allocator& alloc) {
```

Comment:
Expressive name. Perhaps `convert_and_wrap_in_seq` is good enough?

Reply:
Yeah, good call